src/app/parcels/ExpandedParcelDetails.tsx

```diff
@@ -16,7 +16,11 @@ const ExpandedParcelDetails = async ({ parcelId }: Props): Promise<React.ReactEl
     return (
         <>
             <DataViewer data={expandedParcelDetails.expandedParcelData} />
-            <EventTable tableData={expandedParcelDetails.events} />
+            <EventTable
+                tableData={expandedParcelDetails.events.sort(
+                    (eventA, eventB) => eventB.timestamp.getTime() - eventA.timestamp.getTime()
+                )}
```

Comment:
should we have a function for this and call it `sortByTimestampWithMostRecentFirst` or some better name 😆 so the intention is clearer

Reply:
Have done :)